docs/getting-started/ali-function-compute.md

```diff
@@ -0,0 +1,157 @@
+# Alibaba Cloud Function Compute
+
+[Alibaba Cloud Function Compute](https://www.alibabacloud.com/en/product/function-compute) is a fully managed, event-driven compute service. Function Compute allows you to focus on writing and uploading code without having to manage infrastructure such as servers.
+
+## 1. Setup
+
+> [serverless-devs](https://github.com/Serverless-Devs/Serverless-Devs) is an open source and open serverless developer platform dedicated to providing developers with a powerful tool chain system. Through this platform, developers can not only experience multi cloud serverless products with one click and rapidly deploy serverless projects, but also manage projects in the whole life cycle of serverless applications, and combine serverless devs with other tools / platforms very simply and quickly to further improve the efficiency of R & D, operation and maintenance.
+
+Install [serverless-devs](https://github.com/Serverless-Devs/Serverless-Devs) CLI
+
+```sh
+npm install @serverless-devs/s -g
+```
+
+Add the AK & SK configuration
+
+```sh
+s config add 
+# Please select a provider: Alibaba Cloud (alibaba)
+# Input your AccessKeyID & AccessKeySecret
+```
+
+## 2. Hello World
+
+Edit `src/index.ts`:
+
+```ts
+import { serve } from "@hono/node-server";
+import { Hono } from "hono";
+
+const REQUEST_ID_HEADER = "x-fc-request-id";
+
+const app = new Hono();
+
+app.post("/initialize", (c) => {
+  console.log(`RequestId: ${c.req.header(REQUEST_ID_HEADER)}`);
+  return c.text("Initialize");
+});
+
+app.post("/invoke", (c) => {
+  console.log(`RequestId: ${c.req.header(REQUEST_ID_HEADER)}`);
+  return c.text("Invoke");
+});
+
+app.get("/", (c) => {
+  return c.text("Hello from index!");
+});
+
+app.get("/hello", (c) => {
+  return c.text("Hi!");
+});
+
+const port = 9000;
+console.log(`Server is running on port ${port}`);
+
+serve({
+  fetch: app.fetch,
+  port,
+});
+```
+
+Edit `package.json`:
```

Comment:
Here, we need to show only the `scripts` section in `package.json`. I think we can remove `dependencies` and `devDependencies`. If you want to describe dependencies, you can write a command like `npm i -D esbuild` referring to other pages.

Reply:
fixed by <https://github.com/honojs/website/pull/417/commits/7a4ecabd4ead3bac0859af4fc7990ac639eadeb2>